<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalancePrintBase</class>
 <widget class="QDialog" name="BalancePrintBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Impresion del balance de sumas y saldos</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <item>
      <widget class="QToolButton" name="mui_ccostes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../bulmages.qrc">
         <normaloff>:/BulmaCont32x32/images/png/i_centroCoste.xpm</normaloff>:/BulmaCont32x32/images/png/i_centroCoste.xpm</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mui_canales">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../bulmages.qrc">
         <normaloff>:/BulmaCont32x32/images/png/i_canales.xpm</normaloff>:/BulmaCont32x32/images/png/i_canales.xpm</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mui_imprimir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../bulmages.qrc">
         <normaloff>:/BulmaCont32x32/images/png/i_filePrint.xpm</normaloff>:/BulmaCont32x32/images/png/i_filePrint.xpm</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>430</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Datos del listado:</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1">
            <property name="text">
             <string>Codigo inicial
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BlSearchWidget" name="m_codigoinicial"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_3">
            <property name="text">
             <string>Fecha inicial</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BlDateSearch" name="m_fechainicial1"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_2">
            <property name="text">
             <string>Codigo final</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BlSearchWidget" name="m_codigofinal"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_2_2">
            <property name="text">
             <string>Fecha final</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BlDateSearch" name="m_fechafinal1"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_3_2">
            <property name="text">
             <string>Nivel de Cuentas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combonivel"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checksuperiores">
            <property name="text">
             <string>Incluir &amp;niveles superiores.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox1_2">
            <property name="text">
             <string>Incl&amp;uir resultados parciales</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Formato:</string>
       </property>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radiotexto">
          <property name="text">
           <string>&amp;Texto Plano</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radiopropietario">
          <property name="text">
           <string>Propietario</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radiohtml">
          <property name="text">
           <string>HT&amp;ML</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radiokugar">
          <property name="text">
           <string>&amp;Kugar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>BlDateSearch</class>
   <extends>QWidget</extends>
   <header>bldatesearch.h</header>
  </customwidget>
  <customwidget>
   <class>BlSearchWidget</class>
   <extends>QWidget</extends>
   <header>blsearchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages.qrc"/>
  <include location="../../../bulmages.qrc"/>
 </resources>
 <connections/>
</ui>
